tests/tests_unit/test_requests.py

```diff
@@ -1,14 +1,12 @@
-import unittest
-from ..compat import mock
+import mock
 
 
-class RequestsTest(unittest.TestCase):
-
+class TestRequests:
     def test_request_works(self):
         url = '/metaproducts/island/1/1'
         method = 'get'
 
         with mock.patch('mkmsdk.api.Api.request') as mock_request:
             mock_request(url, method)
 
```

Comment:
This test is significantly different (before you were outside the context manager). Is it OK?